assignments/arrays.js

```diff
@@ -63,32 +63,68 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+const vehicle =  inventory.filter((v) => v.id === 33);
+
+
+
+console.log(`Car 33 is a ${vehicle[0].car_year} ${vehicle[0].car_model}` );
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+
+
+function lastcar(){
+    let last = inventory.length - 1;
+    return inventory[last];
+}
+console.log(lastcar());
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+
```

Comment:
Another and shorter method of doing this would be to map of the inventory and the the car models, and then just sort that before console.log()

let carModels = inventory.map(car => {
  return car.car_model;
})
console.log(carModels.sort());